genon/serving/paddle/etc/smoke_test_inference.py

```
@@ -0,0 +1,108 @@
+
+def main() -> None:
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--out", required=True, help="결과 JSON 출력 경로")
+    parser.add_argument(
+        "--image",
+        default=None,
+        help="OCR 대상 이미지 경로. 미지정 시 임시 합성 이미지 사용",
+    )
+    args = parser.parse_args()
+
+    print(f"[smoke] paddle device check ...", flush=True)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)